spec/index.html

```diff
@@ -244,8 +244,10 @@ <h2>A Canonical form of N-Quads</h2>
     less variability in layout.
     The grammar for the language is the same.</p>
 
-  <p class="note">Even when not explicitly serializing
-    canonical N-Quads, implementers are encouraged to produce this form.</p>
+  <p class="note">A canonical form for N-Quads can be used to ensure
+    that the form of a quad is unique for a given choice of
```

Comment:
`the form of a quad` doesn't seem right, if my understanding is correct, that this `form of N-Quads` should result in reduction of _semantically identical but syntactically different_ quads to a single _canonical_ quad...

Reply:
"syntactically different" needs some work. We don't want to imply the writing of the RDF terms themselves is affected.

"presentation"?

Reply:
With the provision that this can't really be fully accomplished until #16 is also considered, the intention of this note is to limit choice of representing code points in the resulting RDF term (literal, in this case).

How about something like the following:

> A canonical form of N-Quads can be used to ensure
> that variations in the syntactic representation of terms
> within that quad is determined; each code point
> can be represented by only one of
> `UCHAR`, `ECHAR` and unencoded character
> where the relevant production allows for a choice in representation.

Reply:
See updated note.